src/lib_ccx/ccx_common_common.c

```diff
@@ -57,14 +57,12 @@ void millis_to_time(LLONG milli, unsigned *hours, unsigned *minutes,
 	milli = (milli - *minutes) / 60; // Remainder, in hours
 	*hours = (int)milli;
 }
+
 /* Frees the given pointer */
-void freep(void *arg)
+void freep(void **arg)
 {
-	void **ptr = (void **)arg;
-	if (*ptr)
-		free(*ptr);
-	*ptr = NULL;
-
+	if (*arg)
+		free(*arg);
```

Comment:
It should be
if (arg)
free (*argc)

Reply:
And remember that this affects the callers which now how to pass a pointer to the pointer they want to free

Reply:
This had already been fixed when you wrote your comment, probably pushed while you were reviewing